src/Services/Twilio/TwilioVerificationService.php

```diff
@@ -18,25 +23,26 @@
 /**
  * @internal
  */
-final class TwilioVerificationService implements PhoneVerificationService
+final class TwilioVerificationService implements PhoneVerificationService, InformsServiceAfterVerification
 {
     public const ERROR_NUMBER_DOES_NOT_SUPPORT_SMS = 60205;
     public const ERROR_NOT_FOUND = 20404;
 
-    private VerificationList $verifications;
-    private VerificationCheckList $verificationChecks;
+    private ServiceContext $twilio;
 
-    public function __construct(Client $twilio, string $verifyId)
-    {
-        $twilio = $twilio->verify->v2->services($verifyId);
-        $this->verifications = $twilio->verifications;
-        $this->verificationChecks = $twilio->verificationChecks;
+    public function __construct(
+        Client $twilio,
+        string $verifyId,
+        private ?VerificationCodeManager $verificationCodeManager,
+        private ?Dispatcher $event,
```

Comment:
I think we should always dispatch the verified event, so let's just make this a requirement 🤔